helpers/ayah/convert_quran_words_ar_csv.py

```diff
@@ -0,0 +1,62 @@
+'''Convert quran_words_ar.csv to django fixtures file'''
+import json
+from os import path
+import csv
+from collections import defaultdict
+
+
+def structure_surah_dict(data):
+    '''Parse raw csv data and build structured dict'''
+    surah_dict = defaultdict(lambda: defaultdict(list))
+    for surah, ayah, position, word in data:
+        if ayah not in surah_dict[surah]:
+            surah_dict[surah][ayah] = []
+
+        if position > len(surah_dict[surah][ayah]):
```

Comment:
This condition doesn't make sense to me, tbh. The bad case if you will see the following two lines:
```
1, 1, 2, word_2
1, 1, 1, word_1
```
In this case you will throw away `word_1` which is incorrect.
Either remove this check completely and assume that the data is sorted by position, or make a proper fix.